<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEntry</class>
 <widget class="QDialog" name="DialogEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Авторизація</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>20</y>
     <width>361</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="9" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item alignment="Qt::AlignBottom">
       <widget class="QPushButton" name="cancel_entry_button">
        <property name="text">
         <string>Відмінити</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignBottom">
       <widget class="QPushButton" name="enter_button">
        <property name="text">
         <string>Увійти</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item alignment="Qt::AlignVCenter">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Ім'я користувача</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignHCenter</set>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignVCenter">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Пароль</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignHCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLineEdit" name="username_field">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="password_field">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Авторизація</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
